docker-compose.replicas-3.yml

```diff
@@ -114,12 +109,6 @@ services:
     environment:
       - MONGO_INITDB_ROOT_USERNAME=${MONGODB_ADMIN_USER}
       - MONGO_INITDB_ROOT_PASSWORD=${MONGODB_ADMIN_PASSWORD}
-      - USER_MGNT_MONGODB_PASSWORD=${USER_MGNT_MONGODB_PASSWORD}
-      - HEARTH_MONGODB_PASSWORD=${HEARTH_MONGODB_PASSWORD}
-      - CONFIG_MONGODB_PASSWORD=${CONFIG_MONGODB_PASSWORD}
-      - METRICS_MONGODB_PASSWORD=${METRICS_MONGODB_PASSWORD}
-      - OPENHIM_MONGODB_PASSWORD=${OPENHIM_MONGODB_PASSWORD}
-      - WEBHOOKS_MONGODB_PASSWORD=${WEBHOOKS_MONGODB_PASSWORD}
```

Comment:
@rikukissa why are all these being removed?  Did they not need to be environment variables in the first place?

Reply:
Yea, exactly. It's enough if `mongo-on-update` knows about them. I think I first had some sort of mongo init script which created the users before the current approach